### ISSUE TITLE
Conflict between multiple mods using ModelReplacementAPI

### ISSUE THREAD

**Issue by FuttBucker3000** [BunyaPineTree/LethalCompany_ModelReplacementAPI#54]
I have two player models that I created with model replacement api on unity but whenever I load the game with both I get this error:

[Info   :   BepInEx] Loading [NoEyelash2B 0.1.0]
[Info   :   Console] nier2bnoeyelashes.templatebundle
[Info   :meow.ModelReplacementAPI] Registering body replacement type ModelReplacement.MR2B to suit name pajamasuit.
[Info   :NoEyelash2B] Plugin com.bonelessmint.NoEyelash2B is loaded!
[Info   :   BepInEx] Loading [Pink Master Chief Player Model 0.1.0]
[Info   :   Console] PinkMasterChief.templatebundle
**[Error  : Unity Log] The AssetBundle 'IO.Stream' can't be loaded because another AssetBundle with the same files is already loaded.**
[Info   :meow.ModelReplacementAPI] Registering body replacement type ModelReplacement.MRHALOCESPARTAN1 to suit name slayer.
[Info   :Pink Master Chief Player Model] Plugin com.bonelessmint.PinkMasterChief is loaded!

Nier2BNoEyelashes being one model and PinkMasterChief being the other.

**Comment by FuttBucker3000** [comment]
I don't know how it prioritizes one or the other, but only one of the two will actually make it into the game. I also had this conflict with a Doom Guy model on thunderstore. 

https://thunderstore.io/c/lethal-company/p/TRIPLEX/Doomguy_model/

**Comment by FuttBucker3000** [comment]
I think I found a fix.
When building the project in Unity, I changed the assetbundle name to something other than templatebundle. 
For example, I changed the assetbundle name to templatebundle2b for the Nier 2B model. Seems to have ended the conflicts since I think the issue was that multiple mods were using the same file name for their templatebundle? I am a super beginner to this stuff, so I just trial and error most of it.
